app/services/scene-collections/nodes/sources.ts

```diff
@@ -261,6 +262,18 @@ export class SourcesNode extends Node<ISchema, {}> {
       if (sourceInfo.hotkeys) {
         promises.push(supportedSources[index].hotkeys.load({ sourceId: sourceInfo.id }));
       }
+
+      this.sourceFiltersService.loadFilterData(
+        sourceInfo.id,
+        sourceInfo.filters.items.map(f => {
+          return {
+            name: f.name,
+            type: f.type,
+            visible: f.enabled,
+            settings: f.settings,
+          };
+        }),
+      );
```

Comment:
is this the fix for media backup filters?

Reply:
This is actually just due to the fact that filters are loaded in a bulk function.  Without this, the store would be out of date, because the sources are being created in bulk by the backend, and the frontend doesn't know about them.  We didn't need this before because we weren't trying to keep an accurate picture of filters in the frontend store before.